db/blob/blob_source.h

```diff
@@ -29,9 +29,8 @@ class Slice;
 // storage with minimal cost.
 class BlobSource {
  public:
-  BlobSource(const ImmutableOptions* immutable_options,
-             const std::string& db_id, const std::string& db_session_id,
-             BlobFileCache* blob_file_cache);
+  BlobSource(const ImmutableOptions* ioptions, const std::string& db_id,
```

Comment:
Any reason for the renaming?

Reply:
Just to make the new code in `BlobSource::BlobSource` more readable -- Keep the assignment in a single line.